btmn_106.nv.json

```diff
@@ -0,0 +1,47 @@
+{
+	"_notes": ["Compiled by Horsepin."],
+	"_copyright": "Copyright (C) 2016 by Tom Collins <tom@tomlogic.com>",
```

Comment:
Feel free to update the copyright to your name/email and the current year.  Or leave it out.  Or say, "Copyright (C) 2017 assigned to Tom Collins <tom@tomlogic.com>".

Reply:
Copy paste gets you in trouble even for a json :)